src/app/components/Landing.tsx

```diff
@@ -33,13 +34,13 @@ const Landing = () => {
 
   return (
     <>
-      {currentUserDB.email && (
-        <>
+      {currentUser ? (
```

Comment:
instead of this condition try to do it before return. Cleaner way of doing that is:
`if(!currentUser) return <Spinner>
`
`
return (rest of JSX)
`